src/kava.js

```diff
@@ -467,7 +469,10 @@ class Kava extends BasePlugin {
       this._timer.start();
       this._isFirstPlay = false;
       this._model.updateModel({
-        joinTime: Kava._getTimeDifferenceInSeconds(this._firstPlayRequestTime)
+        joinTime:
+          this.player.config.playback.preload === 'auto' || this._isManualPreload
```

Comment:
I think `this._isManualPreload` is reliable enough

Reply:
@yairans in cases where there is a low network there is a big chance that event if load api was called, it may hang and take long time until canplay will be emitted. And in that case a user may already press play and a play event will still come before canplay.
In the above case, relying on the `preload` flag will get correct time and the event won't.
This is the correct way to measure it, and it also should be stronger then the `_isManualPreload` check.